blocklib/digitizers/include/gnuradio/digitizers/tags.h

```diff
@@ -67,39 +68,29 @@ namespace gr {
     inline gr::tag_t
     make_acq_info_tag(const acq_info_t &acq_info, uint64_t offset)
     {
-      gr::tag_t tag;
-      tag.key = pmt::intern(acq_info_tag_name);
-      tag.value =  pmt::make_tuple(
-              pmt::from_uint64(static_cast<uint64_t>(acq_info.timestamp)),
-              pmt::from_double(acq_info.timebase),
-              pmt::from_double(acq_info.user_delay),
-              pmt::from_double(acq_info.actual_delay),
-              pmt::from_long(static_cast<long>(acq_info.status))
-              );
-      tag.offset = offset;
-      return tag;
+      return {offset, {{acq_info_tag_name, std::vector<pmtf::pmt>{static_cast<uint64_t>(acq_info.timestamp), static_cast<double>(acq_info.timebase), static_cast<double>(acq_info.user_delay), static_cast<double>(acq_info.actual_delay), static_cast<long>(acq_info.status)}}}};
     }
 
     inline acq_info_t
     decode_acq_info_tag(const gr::tag_t &tag)
     {
-      assert(pmt::symbol_to_string(tag.key) == acq_info_tag_name);
```

Comment:
Any specific reason to remove the asserts ?  I suppose they might be useful if someone tries to decode the wrong tag.